usaspending_api/disaster/management/commands/generate_covid19_download.py

```diff
@@ -100,12 +100,12 @@ def download_file_list(self):
             (
                 sql_dir / "disaster_covid19_file_a.sql",
                 self.working_dir_path
-                / f"{self.get_current_fy_and_period}-Present_All_TAS_AccountBalances_{short_timestamp}",
+                / f"FY2020P07-Present_All_TAS_AccountBalances_{short_timestamp}",
```

Comment:
What ticket do these changes fall under?
The Readme File lists
```
...
FY20P07-Present_All_TAS_AccountBalances_
...
FY20P07-Present_All_TAS_AccountBreakdownByPA-OC_
...```

Reply:
@dpb-bah

Reply:
Ah no ticket has been made as of yet. I can remove these.